tests/integ/modin/groupby/test_groupby_named_agg.py

```diff
@@ -19,8 +22,13 @@ def test_invalid_named_agg_errors(basic_snowpark_pandas_df):
     )
 
 
-@sql_count_checker(query_count=5)
+@sql_count_checker(query_count=6)
 def test_invalid_func_with_named_agg_errors(basic_snowpark_pandas_df):
+    # Temporary workaround until Anaconda uploads the Modin package
```

Comment:
can you add a `pytest.mark.parametrize("custom_package", [True,False])`?

For the setting, you might want to use 

```
value =  pd.session.custom_package_usage_config["enabled"]
try:
   pd.session.custom_package_usage_config["enabled"]
   finally:
 pd.session.custom_package_usage_config["enabled"] = value

as this may affect other tests.

Reply:
Unfortunately I think that pd.session.custom_package_usage_config is unset before this test - for now, let's mark this as an XFAIL with a ticket for after the anaconda packages are released?